sbc_compassion/models/import_letter_line.py

```diff
@@ -92,3 +109,23 @@ def _set_name(self):
                 line.name = str(
                     line.sponsorship_id.partner_codega) + " - " + str(
                         line.child_code)
+
+    @api.multi
+    def save_lines(self, mandatory_review=False):
```

Comment:
`get_letter_datas` would be more appropriate to what the function is doing (it doesn't save anything).